plugin.go

```diff
@@ -169,10 +169,6 @@ func (p Plugin) CheckAllowed(ip string) (allow bool, country string, err error)
 		return false, ip, fmt.Errorf("lookup of %s failed: %w", ip, err)
 	}
 
-	if country == "-" {
-		return p.allowPrivate, country, nil
-	}
-
```

Comment:
With this check removed, the `AllowPrivate` option is no longer used anywhere.

Is the intention to have its behavior replaces with `DefaultAllow`? In that case, could you please remove `AllowPrivate` from the code, and also from the configuration sample in the README?

https://github.com/nscuro/traefik-plugin-geoblock/blob/8542be39eb20ba924e7786bc821d7b8dbc2c7757/README.md?plain=1#L56-L57

Reply:
Sorry for the delay... that certainly looks like an error.  I don't recall the context, now.  We ended up not using this locally, as we had a higher-order problem which was the real culprit of issues we were having.